extensions/amp-story-page-attachment/0.1/amp-story-draggable-drawer.js

```diff
@@ -42,7 +42,7 @@ export const DrawerState = {
 const renderDrawerElement = () => {
   return (
     <div class="i-amphtml-story-draggable-drawer">
-      <div class="i-amphtml-story-draggable-drawer-container">
+      <div class="i-amphtml-story-draggable-drawer-container" hidden>
```

Comment:
Can we do this from the `buildCallback` instead, so if we look for `toggle(` we find everything?